packages/apsara-ui/src/Tag/Tag.tsx

```diff
@@ -23,13 +26,14 @@ const Tag = ({
     closable = false,
     closeIcon,
     icon,
+    size = "medium",
     ...props
 }: TagProps) => {
     const [visible, setVisible] = useState(true);
     if (!visible) return null;
     return (
         <TagWrapper>
-            <StyledTag {...props} type={type} color={color} closable={closable} icon={icon ? true : false}>
+            <StyledTag {...props} type={type} color={color} closable={closable} icon={icon ? true : false} size={size}>
```

Comment:
If we are only changing the font-size using this prop then let's rename it to fontSize to avoid confusion?

Reply:
this new prop is to let user customize the Tag sizing using `small | medium | large` value without having to care about font sizing or other styling. And apparently, this only needs to change the font size because the wrapper box size will also adapt based on the font size.

on the other hand, if user need to customize this component's font size they can use `props.style`